x/crosschain/keeper/grpc_query_cctx_rate_limit_test.go

```diff
@@ -144,13 +366,14 @@ func TestKeeper_ListPendingCctxWithinRateLimit(t *testing.T) {
 		zk.ObserverKeeper.SetTSS(ctx, tss)
 
 		// Set rate limiter flags as disabled
-		rateLimiterFlags := sample.RateLimiterFlags()
-		rateLimiterFlags.Enabled = false
-		k.SetRateLimiterFlags(ctx, rateLimiterFlags)
+		rFlags := sample.RateLimiterFlags()
```

Comment:
```suggestion
		rateLimiterFlags := sample.RateLimiterFlags()
```
That's one another example

rFlags is confusing, you would need to look at the variable definition to understand the code